leveldb-library.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '5.0'
   s.osx.deployment_target = '10.7'
   s.tvos.deployment_target = '10.0'
+  s.watchos.deployment_target = '6.0'
 
   s.source       =  { 
```

Comment:
The line below will need to be hardcoded to 1.22

Reply:
with a comment

Reply:
Thanks!